src/store.ts

```diff
@@ -1,88 +1,84 @@
-    if (fs.existsSync(asarPath)) {
-      return process.nextTick(() => {
-        callback(asarPath);
-      });
-    }
-  }
+	let xpmFolder = path.resolve(__dirname, '..');
+	let appFolder = path.dirname(xpmFolder);
+	if (path.basename(xpmFolder) === 'xpm' && path.basename(appFolder) === 'app') {
+		const asarPath = (appFolder + '.asar');
+		if (fs.existsSync(asarPath)) {
+			return process.nextTick(() => {
+				callback(asarPath);
+			});
+		}
+	}
 
-  xpmFolder = path.resolve(__dirname, "..", "..", "..");
-  appFolder = path.dirname(xpmFolder);
-  if (path.basename(xpmFolder) === "xpm" && path.basename(appFolder) === "app") {
-    let asarPath = (appFolder + ".asar");
-    if (fs.existsSync(asarPath)) {
-      return process.nextTick(() => {
-        callback(asarPath);
-      });
-    }
-  }
+	xpmFolder = path.resolve(__dirname, '..', '..', '..');
+	appFolder = path.dirname(xpmFolder);
+	if (path.basename(xpmFolder) === 'xpm' && path.basename(appFolder) === 'app') {
+		let asarPath = (appFolder + '.asar');
+		if (fs.existsSync(asarPath)) {
+			return process.nextTick(() => {
+				callback(asarPath);
+			});
+		}
+	}
 
-  switch (process.platform) {
-    case "win32": {
-      process.nextTick(() => {
-        const appLocation = path.join(process.env.ProgramFiles, "Xanite", "resources", "app.asar");
-        callback(appLocation);
-      });
-      return;
-    }
-    case "darwin": {
-      child_process.exec(`mdfind "kMDItemCFBundleIdentifier == \'io.github.xanite\'"`, (error, stdout = "", stderr) => {
-        if (!error) {
-          let appLocation = (stdout === "" ? "/Applications/Xanite.app" : stdout.split("\n"));
-          callback((appLocation + "/Contents/Resources/app.asar"));
-        }
-      });
-      return;
-    }
-    case "linux": {
-      let appLocation = "/usr/local/share/xanite/resources/app.asar";
-      if (!fs.existsSync(appLocation)) {
-        appLocation = "/usr/share/xanite/resources/app.asar";
-      }
-      process.nextTick(() => {
-        callback(appLocation);
-      });
-      return;
-    }
-  }
-
-  process.nextTick(() => {
-    callback(null);
-  });
+	switch (process.platform) {
+		case 'win32':
+			process.nextTick(() => {
+				const appLocation = path.join(process.env.ProgramFiles, 'Xanite', 'resources', 'app.asar');
+				callback(appLocation);
+			});
+			break;
+		case 'darwin':
+			child_process.exec(`mdfind 'kMDItemCFBundleIdentifier == \'io.github.xanite\''`, (error, stdout = '', stderr) => {
+				if (!error) {
+					const appLocation = (stdout === '' ? '/Applications/Xanite.app' : stdout.split('\n'));
+					callback((appLocation + '/Contents/Resources/app.asar'));
+				}
+			});
+			break;
+		case 'linux':
+			let appLocation = '/usr/local/share/xanite/resources/app.asar';
+			if (!fs.existsSync(appLocation)) {
+				appLocation = '/usr/share/xanite/resources/app.asar';
+			}
+			process.nextTick(() => {
+				callback(appLocation);
+			});
+			break;
+		default: {
+			callback(null);
+		}
+	}
```

Comment:
Might it be worth turning code like this into a module?  I feel like it would make the source more clean, and be useful outside of xanite.